src/engine/cuecontrol.cpp

```diff
@@ -481,23 +481,23 @@ void CueControl::hotcueActivatePreview(HotcueControl* pControl, double v) {
     if (v) {
         if (pCue && pCue->getPosition() != -1) {
             m_iCurrentlyPreviewingHotcues++;
-            int iPosition = pCue->getPosition();
```

Comment:
Was there a reason we used integer here in the past? Perhaps double is too precise?